tests/runtime/conftest.py

```diff
@@ -98,13 +100,18 @@ def sample_inference_service(render_protos) -> ServicePackage:
 
 @pytest.fixture(scope="session")
 def sample_predict_servicer(sample_inference_service) -> GlobalPredictServicer:
-    servicer = GlobalPredictServicer(inference_service=sample_inference_service)
+    interrupter = ThreadInterrupter()
+    interrupter.start()
```

Comment:
NIT: Would it make sense for the GPS to take ownership over the `start/stop` logic of the `interrupter`? I suppose the lifecycle of an `interrupter` may be shared between multiple servicers, though.

Reply:
Yeah, I thought about that but 
- the current billing metric start/stop is a bit awkward
- the interrupting only makes sense in the context of a running server 
- the servers each own their own GPS
- and there may be multiple servicers like the ModelRuntimeServicer, like you said